modules/markdown/markdown.js

```diff
@@ -15,7 +15,7 @@ import {Link} from './link'
 import {Image} from './image'
 import {List, ListItem} from './list'
 
-const Softbreak = () => <Text>{'\n'}</Text>
+const Softbreak = () => ' '
 const Hardbreak = () => <Text>&lt;br&gt;</Text>
```

Comment:
I suppose we could go ahead and render hard breaks (lines ending with two spaces) as newlines now.

Reply:
If you wanted.

Reply:
Inline HTML has nothing to do with hard linebreaks, fwiw. 

(Embedding literal HTML in the markdown source is broken; _I_ chose to print an escaped br tag here, when I was seeing what things did, and then I never encountered a hard linebreak to cause me to change it.)

But OK.

Reply:
You're right. Thanks. I'll follow up with an additional change.